### ISSUE TITLE
MP: Chamber of bones

### ISSUE THREAD

**Issue by AJenbo** [diasurgical/devilutionX#5927]
Door to the chamber is already open when entering the level

**Comment by obligaron** [comment]
How can the issue be reproduced?

New game:
![grafik](https://user-images.githubusercontent.com/25415264/229347696-24cb23d0-d2d0-4136-9083-c9091a1606d0.png)
Or is this the wrong door?

While testing I only saw that sometimes the skeletons opens the door but don't leave the room. Sadly this also hard to reproduce.

**Comment by AJenbo** [comment]
It's the right door, I'm unsure how to replicate it we just entered the level and monsters where charging out of there in hords.

We did skip reading the book, instead teleporting to the entrance, but i don't think that has any effect. I was the first to enter, the one of the other players and lastly the host.

**Comment by obligaron** [comment]
Tested again. I got a rhino to open the door for me.
- Teleported to the door.
- Rhino can't walk straight to me, cause other monsters surround me.
- He could walk to me through the door tile
- That's why the rhino opens the door and walks to me.
- when he tries to walk to me through the door tile.

![grafik](https://user-images.githubusercontent.com/25415264/229358395-13fdaac4-fd05-481a-9c3f-cb5baa170a93.png)

Could this also happen in your case? 🤔
